src/js/common/utils/helpers.js

```diff
@@ -101,14 +101,33 @@ export function dateToMoment(dateField) {
   });
 }
 
+export function dropTimeAndTimezone(date) {
+  let newDate = moment(date).format();
+  const indexOfTime = newDate.indexOf('T');
+  if (indexOfTime >= 0) {
+    newDate = newDate.substring(0, indexOfTime);
+  }
+  return newDate;
+}
+
 export function formatDateLong(date) {
   return moment(date).format('MMMM DD, YYYY');
 }
 
+export function formatDateWithoutTimeLong(date) {
```

Comment:
Fair enough.  I would still question the need for two functions `formatDateWithoutTimeLong` & `formatDateLong` implying that in some places we _do_ and in other places we _do not_ want timezone incorporated into the formatting of the date.